content/pt/docs/guias/APROVACAO/Aprovando-uma-transacao/_index.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Aprovando uma Transação"
+linkTitle: "Aprovando uma Transação"
+date: 2021-06-28T09:00:00-03:00
+lastmod: 2021-06-28T09:00:00-03:00
+weight: 2
+draft: false
+description: >
+
+---
+<br>
+
+Caso uma transação tenha sido iniciada por uma plataforma terceira em uma conta Stone, oferecemos algumas formas ao usuário para que o mesmo possa avaliar e realizar a aprovação ou reprovação das transações. Verifique o melhor formato para seu negócio abaixo:
+
+##### **<ins>Aplicativo Aprovador</ins>**
```

Comment:
hey @agabiandrade, why have you opted to use underlined titles? it decreases legibility and really seems that this is a link to something. What do you think?

Reply:
It makes sense! Updated!